style/components/questioncard.css

```
@@ -25,7 +25,7 @@ article .material-symbols-outlined {
   background-color: transparent;
 }
 
-article .material-symbols-outlined.not-selected {
+article .material-symbols-outlined.card__bookmark--not-active {
```

I don't think that you need the type selector `article` here; it should be sufficient to use
```suggestion
.card__bookmark--active {
```
and above, you change `article .material-symbols-outlined` into `.card__bookmark` and switch the styles for the **not active bookmark**.